image/templates/helm/stackrox-secured-cluster/templates/admission-controller.yaml

```diff
@@ -60,9 +60,11 @@ spec:
       nodeSelector:
         {{- ._rox.admissionControl._nodeSelector | nindent 8 }}
       {{- end}}
+      {{- if not ._rox.env.openshift }}
```

Comment:
Is it intended to leave these settings for non-OpenShift platforms? Is it because in OpenShift, an SCC is responsible for filling in "good" values, but without an SCC, omitting these would mean to run as root?

Reply:
I am not fully sure what would be the consequence of removing this for vanilla k8s clusters, so I felt safer leaving it as it is. I will give it a quick read maybe to learn what this change would mean for non-OpenShift clusters.